framework/sac_rlhf.py

```diff
@@ -253,7 +255,40 @@ def load_replay_buffer(replay_buffer: ReplayBuffer, path: str):
     replay_buffer.ground_truth_rewards = buffer_data["ground_truth_rewards"]
     replay_buffer.dones = buffer_data["dones"]
 
-    logging.debug(f"Replay buffer loaded from {path}")
+    logging.info(f"Replay buffer loaded from {path}")
+
+
+def uniform_sampling(rb: ReplayBuffer, reward_net: RewardNet):
```

Comment:
Consider moving those three files into a seperate file (e.g. sampling.py) to keep the main file clean.